rct229/ruleset_functions/baseline_systems/baseline_system_util.py

```diff
@@ -147,3 +151,104 @@ def find_exactly_one_fluid_loop(rmi, loop_id):
         loop_id,
         rmi,
     )
+
+
+def has_heating_system(rmi_b, hvac_b_id):
+    """
+    Check whether heating system exists or not.
+
+    Parameters
+    ----------
+    rmi_b json
+        To evaluate if the hvac system is modeled as Sys-2 in the B_RMI.
```

Comment:
This documentation for `rmi_b` must be a holdover from somewhere else.  I think it should be something like
```
rmi_b json
    A ruleset model instance for a baseline RMD.
```

Reply:
Fixed.